charmhelpers/contrib/openstack/utils.py

```diff
@@ -1841,3 +1841,34 @@ def series_upgrade_complete(resume_unit_helper=None, configs=None):
         configs.write_all()
         if resume_unit_helper:
             resume_unit_helper(configs)
+
+
+def run_in_apache(package, release=None):
+    """Return true if given service is run under apache2 with mod_wsgi in
+    this release.
+    Package name is important to lookup version and openstack release
+    """
+
+    if snap_install_requested():
+        return False
+
+    release = release or os_release(package)
+
+    if (package == 'keystone' and
+            CompareOpenStackReleases(release) >= 'liberty'):
+        return True
+
+    if (package == 'cinder-api' and
+            CompareOpenStackReleases(release) >= 'ocata'):
+        return True
+
+    if (package == 'glance-api' and
```

Comment:
I don't think glance API is run under apache2. Can we drop this and then it's ready to merge from my pov.

Reply:
I removed this